nucypher/config/characters.py

```diff
@@ -254,6 +254,7 @@ def static_payload(self) -> dict:
         payload = dict(provider_uri=self.provider_uri,
                        poa=self.poa,
                        light=self.is_light,
+                       domains=list(self.domains),  # FIXME: 1496
```

Comment:
Does this PR close this issue?

Reply:
Yes, it does. I have to clean and clarify all those FIXMEs and TODOs. Perhaps open a better scoped issue, if needed.

Reply:
✅